code/game/turfs/simulated.dm

```diff
@@ -17,7 +17,7 @@
 	if(wet >= wet_setting)
 		return
 	wet = wet_setting
-	if(wet_setting == TURF_WET_WATER)
+	if(wet_setting == TURF_WET_WATER || TURF_WET_LUBE)

```

Comment:
Just curious, what exactly is the benefit of changing it to either of these?
The code should work the way I've done it already.